test/functional/NativeTest/nativeTestSettings.mk

```diff
@@ -0,0 +1,43 @@
+
+##############################################################################
+#  Copyright (c) 2020, 2020 IBM Corp. and others
+
+#  This program and the accompanying materials are made available under
+#  the terms of the Eclipse Public License 2.0 which accompanies this
+#  distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+#  or the Apache License, Version 2.0 which accompanies this distribution and
+#  is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+#  This Source Code may also be made available under the following
+#  Secondary Licenses when the conditions for such availability set
+#  forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+#  General Public License, version 2 with the GNU Classpath
+#  Exception [1] and GNU General Public License, version 2 with the
+#  OpenJDK Assembly Exception [2].
+#
+#  [1] https://www.gnu.org/software/classpath/license.html
+#  [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+#  SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+##############################################################################
+
+ # if JDK_VERSION is 8
+ifeq (8, $(JDK_VERSION))
+ TEST_JAVA=$(REPORTDIR_NQ)/testJava/jre/
```

Comment:
Usually, we don't have the last `/` in a variable (i.e., TEST_JAVA). We add `/` when you use it like this: $(TEST_JAVA)/bin/j9vm. So it's more readable.